data/json/overmap/overmap_special/specials.json

```
@@ -5328,5 +5328,36 @@
     "city_sizes": [ 1, 20 ],
     "occurrences": [ 0, 0 ],
     "flags": [ "CLASSIC", "UNIQUE" ]
+  },
+  {
+    "id": "airliner_crashed",
+    "type": "overmap_special",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "airliner_1a_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "airliner_2a_north" },
+      { "point": [ 2, 0, 0 ], "overmap": "airliner_3a_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "airliner_1b_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "airliner_2b_north" },
+      { "point": [ 2, 1, 0 ], "overmap": "airliner_3b_north" },
+      { "point": [ 0, 2, 0 ], "overmap": "airliner_1c_north" },
+      { "point": [ 1, 2, 0 ], "overmap": "airliner_2c_north" },
+      { "point": [ 2, 2, 0 ], "overmap": "airliner_3c_north" },
+      { "point": [ 0, 3, 0 ], "overmap": "airliner_1d_north" },
+      { "point": [ 1, 3, 0 ], "overmap": "airliner_2d_north" },
+      { "point": [ 2, 3, 0 ], "overmap": "airliner_3d_north" },
+      { "point": [ 0, 4, 0 ], "overmap": "airliner_1e_north" },
+      { "point": [ 1, 4, 0 ], "overmap": "airliner_2e_north" },
+      { "point": [ 2, 4, 0 ], "overmap": "airliner_3e_north" },
+      { "point": [ 1, 0, 1 ], "overmap": "airliner_2a_1_north" },
+      { "point": [ 1, 1, 1 ], "overmap": "airliner_2b_1_north" },
+      { "point": [ 1, 2, 1 ], "overmap": "airliner_2c_1_north" },
+      { "point": [ 1, 1, -1 ], "overmap": "airliner_2b_-1_north" },
+      { "point": [ 1, 2, -1 ], "overmap": "airliner_2c_-1_north" }
+    ],
+    "locations": [ "wilderness" ],
+    "city_distance": [ 6, -1 ],
+    "city_sizes": [ 0, 12 ],
+    "occurrences": [ 0, 4 ],
```

With this value it could spawn up to 4 crashed airliners on an overmap. That's a LOT.
```suggestion
    "occurrences": [ 0, 1 ],
```